run_uh.py

```diff
@@ -121,22 +121,18 @@ def excepthook(exception_type, value, tb):
 		with open(outfilename, 'a') as f:
 			traceback.print_exception(exception_type, value, tb, file=f)
 		traceback.print_exception(exception_type, value, tb)
-		print('')
-		print(T('Unknown Horizons has crashed.'))
-		print('')
+		print('\n', T('Unknown Horizons has crashed.', '\n'))
 		print(T('We are very sorry for this and want to fix the underlying error.'))
 		print(T('In order to do this, we need the information from the logfile:'))
 		print(outfilename)
 		print(T('Please give it to us via IRC or our forum, for both see http://unknown-horizons.org .'))
 	return excepthook
 
-def exithandler(exitcode, signum, frame):
+def exithandler(exitcode):
```

Comment:
This change is wrong, `exithandler` needs to accept three arguments (and will be passed three arguments).

The `signal.signal` function expects a handler function that accepts two arguments: signum and frame. See here: https://docs.python.org/2/library/signal.html#signal.signal

It might be a bit confusing though, I can understand that. The `exithandler` function isn't called by us, but by the `signal` module.

```python
signal.signal(signal.SIGINT, functools.partial(exithandler, 130))
```

What `functools.partial(exithandler, 130)` does is return a new function that will pass through all arguments to `exithandler`, but pass 130 as first argument additionally.